week-1/L-functions-nested/exercise.js

```diff
@@ -1,5 +1,17 @@
-var mentor1 = "Daniel";
-var mentor2 = "Irina";
-var mentor3 = "Mimi";
-var mentor4 = "Rob";
-var mentor5 = "Yohannes";
+var student = 104;
+var mentor = 56;
+var total = student + mentor;
+var percentMentor = Math.round((mentor * 100) / total);
+var percentStudent = Math.round((student * 100) / total);
+function message() {
+  var var1 =
+    "percentage of mentors is " +
+    percentMentor +
+    "%" +
+    " and percentage of student is  " +
+    percentStudent +
+    "%";
+
+  return var1;
```

Comment:
As a general rule, variable names are *nouns* and function names include a *verb*.
How can you renames the variables and `message()` function here to make them clearer and adhere to this rule?